src/main/java/core/basesyntax/db/operations/OperationBalance.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.db.operations;
+
+import core.basesyntax.db.dao.StorageDao;
+import core.basesyntax.model.Item;
+
+public class OperationBalance implements DataOperation {
+    @Override
+    public void handle(Item item, StorageDao dao) {
```

Comment:
Inject StorageDao into impl class
```suggestion
    public void handle(ItemTransaction transaction) {
```